jupyter-book/multi_omics/citeseq_pp.ipynb

```diff
@@ -0,0 +1,2165 @@
+{
```

Comment:
<p>normalise -&gt; normalize (nitpick)</p><p> </p><p>I would explicitly name this "ambient contamination" or something. Please look at what the OSCA book writes. They do this very nicely.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>